setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup, find_packages
+import validator
+
+
+setup(
+    name='content-validator',
```

Comment:
Seems too generic right? `pip install content-validator` can be confusing, we should agree on a standard prefix for Libero Python libraries?

Reply:
`libero-content-validator`? Let's ask in Slack

Reply:
https://libero-community.slack.com/archives/CC6T8DFDW/p1534771747000100

Reply:
``libero-content-validator`` looks like a good option for now. We can change it later if required. No need to distribute this to pypi yet, we can just install it from github to wait for naming feedback.